boards/arm/nucleo_h745zi_q/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+# SPDX-License-Identifier: Apache-2.0
+
+if(CONFIG_PINMUX)
+zephyr_library()
+zephyr_library_sources(pinmux.c)
+zephyr_library_include_directories(${ZEPHYR_BASE}/drivers)
```

Comment:
could you please indent those lines.

Reply:
@tejlmand, indentation is missing in all boards/*/*/CMakeLists.txt I've checked.
Can this be done in a follow up PR to align all boards on this request ?

Reply:
@tejlman I hope this small stuff will not block ma PR, since @erwango already suggested a wider spreader solution.

Reply:
fine to do in followup pr